packages/fxa-auth-server/lib/subhub/client.js

```diff
@@ -196,7 +196,7 @@ const client = function(log, config, statsd) {
             plan_id: isA.string().required(),
           },
           response: isA.alternatives(
-            validators.subscriptionsSubscriptionListValidator,
+            validators.subscriptionsSubscriptionValidator,
```

Comment:
This pairs up with [this subhub swagger fix](https://github.com/mozilla/subhub/pull/394)